phpBB/styles/prosilver/template/ucp_notifications.html

```diff
@@ -31,7 +31,7 @@
 								<!-- IF notification_types.EXPLAIN --><br />&nbsp; &nbsp;{notification_types.EXPLAIN}<!-- ENDIF -->
 							</td>
 							<!-- BEGIN notification_methods -->
-								<td class="mark"><input type="checkbox" name="{notification_types.TYPE}_{notification_methods.METHOD}"<!-- IF notification_methods.SUBSCRIBED --> checked="checked"<!-- ENDIF --> /></td>
+								<td class="mark"><input type="checkbox" name="{notification_types.TYPE}_{notification_types.notification_methods.METHOD}"<!-- IF notification_types.notification_methods.AVAILABLE and notification_types.notification_methods.SUBSCRIBED --> checked="checked"<!-- ENDIF --><!-- IF not notification_types.notification_methods.AVAILABLE --> disabled="disabled"<!-- ENDIF --> /></td>
```

Comment:
I believe it must be `notification_methods.AVAILABLE" only? Or was it just wrong before?

Reply:
It used the values of the header row before, which worked coincidentally. `notification_types.notification_methods.METHOD` is the actual row's value. https://wiki.phpbb.com/Template.assign_block_vars#Example_.232_Nested_blocks

Reply:
Ah, that makes sense!